pkgs/applications/networking/instant-messengers/pidgin/default.nix

```diff
@@ -29,11 +29,13 @@ let unwrapped = stdenv.mkDerivation rec {
 
   NIX_CFLAGS_COMPILE = "-I${gst_all_1.gst-plugins-base.dev}/include/gstreamer-1.0";
 
-  buildInputs = [
+  buildInputs = let
+    python-with-dbus = python.withPackages (pp: with pp; [ dbus-python ]);
```

Comment:
```suggestion
    python = python.withPackages (pp: with pp; [ dbus-python ]);
```

Reply:
It's used just once, so we wouldn't even need to name it, i.e. we could just put it there directly without a let-binding.

Reply:
It is probably cleaner when we name it but I don't see a reason to give it a special name.

Reply:
Ah, but your suggestion won't work due to infinite recursion (`let` is always recursive).